### PR TITLE
feat: add user model; get user when the app starts

### PR DESCRIPTION
**Pull request by Hiroki-Minami**

### CODE REVIEW

File: Chatwis/Models/Language.swift
Comment (itzderr) [inline Chatwis/Models/Language.swift:1]: were you referring to mandarin Chinese?

We could just say Chinese

Comment (Hiroki-Minami) [reply Chatwis/Models/Language.swift:1]: Sure. I'll go with Chinese instead of Mandarin

Comment (Hiroki-Minami) [reply Chatwis/Models/Language.swift:1]: I fixed this.

File: Chatwis/Models/ScreenMode.swift
[the remark this answers is not in the document]
Comment (Hiroki-Minami) [reply Chatwis/Models/ScreenMode.swift:1]: I fixed this.

**Comment by Hiroki-Minami** [comment]
@itzderr 
I finished fixing all your comments. Could you check them again?

**Review by itzderr — APPROVED** [review]

**Pull request merged**